app/presenters/search_result_presenter.rb

```diff
@@ -0,0 +1,2 @@
+class SearchResultPresenter < BasePresenter
+end
```

Comment:
I think after the https://github.com/moneyadviceservice/site_search/pull/2 you can remove this class.